CMakeLists.txt

```diff
@@ -1,64 +1,10 @@
-#
-# If the user specifies -DCMAKE_BUILD_TYPE on the command line, take
-# their definition and dump it in the cache along with proper
-# documentation, otherwise set  CMAKE_BUILD_TYPE to Debug prior to
-# calling PROJECT() 
-#
-
-# IF(DEFINED CMAKE_BUILD_TYPE)
-#    SET(CMAKE_BUILD_TYPE ${CMAKE_BUILD_TYPE} CACHE STRING "Choose the type of
-# build, options are: None(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) Debug
-# Release RelWithDebInfo MinSizeRel.")
-#    MESSAGE(STATUS "Build Type from command line " ${CMAKE_BUILD_TYPE})
-# ELSE()
-#    SET(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build,
-# options are: None(CMAKE_CXX_FLAGS or CMAKE_C_FLAGS used) Debug Release
-# RelWithDebInfo MinSizeRel.")
-#    MESSAGE(STATUS "Default Build Type: Release  (change with
-#    -DCMAKE_BUILD_TYPE=Debug | Release | RelWithDebInfo | MinSizeRel")
-# ENDIF()
-
- 
 cmake_minimum_required (VERSION 2.8.11)
 set(PROJECT_NAME_STR grppi)
 project(${PROJECT_NAME_STR})
 
-
-# Define a path for modules before accesing default. Needed to find TBB
-#set( CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR} )
-# Modified because cmake.modules doesnt exist in the project directory
+# Define path for TBB modules.
 set( CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake.modules ${PROJECT_SOURCE_DIR} )
```

Comment:
This line is used to load the file findTBB.cmake so cmake can find it in the system.